src/components/Dropdown/__stories__/dropdown.stories.scss

```diff
@@ -37,4 +37,8 @@
     align-items: center;
     justify-content: center;
   }
+
+  &_button {
```

Comment:
My two cents about our code examples - they should be copy-pastable into the user's code.
In that case, className is not something that can be easily copied.
I'd suggest using `Flex`'s `gap` and maybe `Box` if it's something that's really necessary.
We can also wrap with our internal example presentation components that know how to arrange elements as a proper example. Then, when the user will copy the example, he can simply delete it.

Reply:
Yeah, but this is not really an example we expect user to copy